config.go

```diff
@@ -90,16 +89,16 @@ func initConfig() error {
 	// Parse XUD config for information about how to connect to the LNDs
 	_, err := toml.DecodeFile(cfg.Xud.Config, &xudCfg)
 
-	if err == nil {
-		if !xudCfg.LndBtc.Disable {
-			setXudLndDefaultValues(xudCfg.LndBtc, true)
-		}
+	if err != nil {
+		return err
+	}
+	// TODO: if LndBtc or LndLtc are disable - you better fail the bot.
```

Comment:
Why should the bot fail to start? The channel manager is optional and can be disabled. The trading bot should still work even if the LNDs are not configured.

Reply:
This is a TODO. No need to do now.

Agree that the bot should fail only if channel manager is enabled and LNDs are disabled. I consider this a wrong setup .

I don't see a good use case for the bot to be used without LNDs. Actually I don't see a good use case for XUD without LND.